misc-utils/uuidd.c

```diff
@@ -57,8 +57,13 @@
 
 #include "nls.h"
 
-/* length of binary representation of UUID */
-#define UUID_LEN	(sizeof(uuid_t))
+/* Protocol segment lenghts */
+enum {
+	UUID_OP_LEN = sizeof(uint8_t),		/* client operation field */
+	UUID_NUM_LEN = sizeof(int),		/* number of requested uuids */
+	UUID_DATA_LEN = sizeof(uuid_t),		/* binary representation of UUID */
+	UUID_BUF_LEN = 1024			/* client - server buffer size */
+};
```

Comment:
I don't think use UUID_ prefix is good idea in this case, it seems like someting releated to the uuid_t (or so).
What about:
```
UUIDD_PROT_OPER_LEN
UUIDD_PROT_NUM_LEN
UUIDD_PROT_DATA_LEN
UUIDD_PROT_BUFSZ
```

Reply:
Maybe to be sure about sizes
```
typedef int32_t uuidd_prot_num_t
typedef uint_t8 uuidd_prot_op_t
```
and in code use
```
   uuidd_prot_num_t num;
   ...
   memcpy(buf, num,  sizeof(num));
```
and _OPER_LEN and _NUM_LEN will be almost unnecessary.

Reply:
Really good  proposal. Thank you @karelzak The change is close to completely rewrote, see b4911618195b981db74d76f8254ab0df1b3ae850.